web/impact/impact/tests/test_user_detail_view.py

```diff
@@ -671,7 +670,6 @@ def test_get_user_with_no_created_at_in_prgs_handled(self):
             person=user,
             program_role__program__program_status=ACTIVE_PROGRAM_STATUS,
             program_role__program__program_family__name="Quigville",
```

Comment:
no need to specify a program_family__name here, please remove

Reply:
Done